contents/blog/web-analytics-mid-mortem.md

```diff
@@ -0,0 +1,75 @@
+---
+date: 2024-02-28
+title: "Mid-mortem: How is the web analytics beta going so far?"
```

Comment:
```suggestion
title: "How to reinvent the wheel: What we've learned from building web analytics"
```

Reply:
I prefer the second, but don't love either atm. We've not really articulated these into specific lessons atm.

Reply:
I don't like the second. Given this is mainly for our users, it could just as easily be "An update on our website analytics beta".

Overall, I think this falls between being about why we're doing it, what we've learned, and updating users, but doesn't totally satisfy any of them. If it's an update, what's happening next + how far are we from launching? If it's what we've learned, why should the reader care?